dabl-meta.yaml

```diff
@@ -13,17 +13,11 @@ catalog:
     icon_file: slack-icon.svg
     source_url: https://github.com/digital-asset/daml-dit-integration-slack
 integration_types:
-    - id: com.projectdabl.integrations.slack_receive_dm
-      name: Slack Receive Direct Message
-      description: Allows direct messages sent to a Slack application to be recorded on a ledger.
-      entrypoint: core_int.integration_slack_receive_dm:integration_slack_receive_dm_main
-      runtime: python-direct
-      fields: []
-    - id: com.projectdabl.integrations.slack_send_message
-      name: Slack Send Message
-      description: Allows outbound messages recorded on a ledger to be sent to Slack
-      entrypoint: core_int.integration_slack_send_message:integration_slack_send_main
-      env_class: core_int.integration_slack_send_message:IntegrationSlackSendMessageEnv
+    - id: com.projectdabl.integrations.slack_integration
+      name: Slack Integration
+      description: Allows outbound messages recorded on a ledger to be sent to Slack, and direct messages sent to a Slack application to be recorded on a ledger.
```

Comment:
copy/pasta 😄

Reply:
"Allows messages to be sent to and received from Slack."